src/domain/data-decoder/entities/schemas/data-decoded.schema.ts

```diff
@@ -0,0 +1,31 @@
+import { Schema } from 'ajv';
+
+export const dataDecodedParameterSchema: Schema = {
+  type: 'object',
+  properties: {
+    name: { type: 'string' },
+    param_type: { type: 'string' },
```

Comment:
The name of this parameter should be `type`. However the CGW renames it to `paramType` when serialising to the clients.

Reply:
Changed in [50ee6e966f4925eeba8f74f82c55e3ff8a216c16](https://github.com/5afe/safe-client-gateway-nest/pull/132/commits/50ee6e966f4925eeba8f74f82c55e3ff8a216c16)

Reply:
Hmm, I'm thinking we should change `paramType` when serialising as well then, right @fmrsabino ? What do you think about opening a new issue to implement the first serializer? It could be decisions to make there as well.

Reply:
> Hmm, I'm thinking we should change `paramType` when serialising as well then, right @fmrsabino ? What do you think about opening a new issue to implement the first serializer? It could be decisions to make there as well.

We should keep using `paramType` for the clients. For now I would just map this payload to the OpenApi entity (just simple instantiation at the route level)

Reply:
I mapped it here [afdc03e954157b284956f02a15792f43bf8a70fd](https://github.com/5afe/safe-client-gateway-nest/pull/132/commits/afdc03e954157b284956f02a15792f43bf8a70fd)

I mapped it inline into `src/routes/data-decode/data-decoded.service.ts` for now, a more elegant solution could be developed when we integrate or response types with frontend DTOs I think.